app/server/models/user.py

```diff
@@ -561,7 +562,7 @@ def user_details(self):
         # should drop the country code from phone number?
         return "{} {} {}".format(self.first_name, self.last_name, self.phone)
 
-    def get_most_relevant_transfer_usage(self):
+    def get_most_relevant_transfer_usages(self):
```

Comment:
now this gets 20 most recent transfers for the user, then creates a map of those usages to the count in this method